### ISSUE TITLE
GLTFLoader: Incorrect bounds for glTF models with normalized positions

### ISSUE THREAD

**Issue by donmccurdy** [mrdoob/three.js#21553]
**Repro**

Attached, a version of `WaterBottle.glb` quantized by my (WIP) quantization implementation in https://github.com/donmccurdy/glTF-Transform/pull/59. The model has a number of validation issues that I'm still working on, but it should be able to render in three.js now, and doesn't.

[WaterBottleQuantized.zip](https://github.com/BabylonJS/Babylon.js/files/6238605/WaterBottleQuantized.zip)

If you load the model and then erase the default bounding box information, it works normally.

**Screenshot:**

![Screen Shot 2021-03-31 at 11 13 38 AM](https://user-images.githubusercontent.com/1848368/113191242-26072080-9212-11eb-8a48-5a85e7d1d4c2.png)

**Desktop (please complete the following information):**

All.

**Smartphone (please complete the following information):**

All.

**Additional context**

Referring to the code here...

https://github.com/mrdoob/three.js/blob/51afad97c9967e9fc4a9bbc3411067320eb835cc/examples/js/loaders/GLTFLoader.js#L2860-L2875

... it looks like three.js (as well as babylon.js, actually) is using the POSITION attribute's `.min` and `.max` property to infer bounds of a mesh primitive, without a more costly traversal of the vertex data. However, the glTF specification states:

> Exporters and loaders must treat these values as having the same data type as accessor's componentType, i.e., use integers (JSON number without fractional part) for integer types and use floating-point decimals for 5126 (FLOAT).

In other words, a normalized accessor will contain normalized integer values in its min/max, and the implementation needs to convert that to floating-point decimals to use as mesh bounds. This becomes an issue for models using `KHR_materials_quantization`, which may choose to use normalized 8-bit or 16-bit integer storage.

**Comment by donmccurdy** [comment]
I was a bit surprised to find this implemented the same way, apparently incorrectly, in both three.js and babylon.js — @lexaknyazev could you confirm I'm understanding the spec right here?

**Comment by zeux** [comment]
I believe the core spec already covers this as a generalized topic, as it's merely a reflection of what value is stored in min/max:

```
accessor.max
Maximum value of each component in this attribute. Array elements must be treated as having the same data type as accessor's componentType. Both min and max arrays have the same length. The length is determined by the value of the type property; it can be 1, 2, 3, 4, 9, or 16.

normalized property has no effect on array values: they always correspond to the actual values stored in the buffer. When accessor is sparse, this property must contain max values of accessor data with sparse substitution applied.
```
